core/src/toga/images.py

```diff
@@ -19,32 +20,34 @@
 # Make sure deprecation warnings are shown by default
 warnings.filterwarnings("default", category=DeprecationWarning)
 
-ImageT = TypeVar("ImageT")
+if TYPE_CHECKING:
+    if sys.version_info < (3, 10):
+        from typing_extensions import TypeAlias, TypeVar
+    else:
+        from typing import TypeAlias, TypeVar
+
+    # Define a type variable for generics where an Image type is required.
+    ImageT = TypeVar("ImageT")
```

Comment:
This is imported in one place, and added to the spelling wordlist, but It doesn't looks like it's actually being used.

Reply:
It was pre-existing; it's used in all the `as_image()` (canvas, image view) and `as_format()` (Image) calls. 

It's in the dictionary now because the default sphinx handling of type annotations apparently considers the type to be a word that needs to be spell checked. The previous plugin didn't do this.